packages/ui/src/routes/+page.svelte

```diff
@@ -10,13 +10,59 @@
 	import { posts } from '$lib/stores/post'
 	import { profile } from '$lib/stores/profile'
 	import { goto } from '$app/navigation'
+	import { browser } from '$app/environment';
 	import Masonry from '$lib/masonry.svelte'
 
-	let y: number
+	let width: number = browser ? window.innerWidth : 0
+	let newColWidth: string
+
+	function checkWidth(w: number) {
```

Comment:
I would rename the function to better describe what it does e.g. `getMasonryColumnWidth` . Similarly, the argument `w` should be a bit more descriptive e.g. `windowWidth`